src/main/java/de/lh/tool/util/mappings/JsonDeserializers.java

```diff
@@ -0,0 +1,25 @@
+package de.lh.tool.util.mappings;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+public abstract class JsonDeserializers {
```

Comment:
Die Klasse dient ja nur als Namespace, oder?
Fände in dem Fall final + private-Konstruktor solider.